### PR TITLE
chore(release): 4.1.0

### PR DESCRIPTION
**Pull request by satish-ravi**

### CODE REVIEW

**Review by jophish — APPROVED** [review]
"technically" a breaking change i think, since a fiat type was removed (replaced, really,) but i'm indifferent towards making lots of major bumps at the moment.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,19 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-### [4.0.0](https://github.com/fiatconnect/fiatconnect-types/compare/v3.3.0...v4.0.0) (2022-05-25)
+## [4.1.0](https://github.com/fiatconnect/fiatconnect-types/compare/v3.3.0...v4.1.0) (2022-06-06)
```
Comment (cajubelt) [inline CHANGELOG.md:5]:
```suggestion
## [4.1.0](https://github.com/fiatconnect/fiatconnect-types/compare/v4.0.0...v4.1.0) (2022-06-06)
```

Comment (satish-ravi) [reply CHANGELOG.md:5]: good catch. Not sure why semantic-release used 3.3.0 here

**Comment by satish-ravi** [comment]
@jophish good point. Would've been nice to start at 0.1.0 for this repo similar to the SDK. I think I'll keep this minor unless anyone feels strongly

File: CHANGELOG.md (hunk 1 above)
Comment (satish-ravi) [reply CHANGELOG.md]: I think this was because v4.0.0 was tagged on a commit on a non-main branch. I'll fix it when I publish this

**Review by cajubelt — APPROVED** [review]

**Pull request merged**